### PR TITLE
removed distro artifacts and corrected changed urls to snapshot downloads

### PR DESCRIPTION
**Pull request by kaikreuzer**
Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

**Comment by kaikreuzer** [comment]
This should fix https://ci.openhab.org/view/Integration%20Builds%20(2.5.x)/job/openHAB2.5.x-linuxpkg/

File: build.gradle [hunk 1]
```diff
@@ -81,15 +81,6 @@ ospackage {
 
 def distributions = [
   [
-    "dist":        'openhab2-release',
```
Comment (BClark09) [inline build.gradle]: Does it make sense to remove them all together? What if someone wants to build their own version of 2.5.1 distro?

We can just change the job for 2.5.x builds on Jenkins.

**Comment by BClark09** [comment]
Thanks Kai! Just a minor comment from me. I will have time to merge and sort out the master branch of repo in preparation early this week.

File: build.gradle (hunk 1 above)
Comment (kaikreuzer) [reply build.gradle]: Yes, it imho makes sense for the HEAD of the 2.5.x branch. For building 2.5.1 distros, we should have a 2.5.1 tag at the appropriate version of the code that can be used for building that version.

Comment (BClark09) [reply build.gradle]: OK, makes sense I think. I need to find a fix for #154 before I can make a 2.5.1 tag for building the distro, but from what you're saying I will do this on master before making changes for 3.x

**Comment by BClark09** [comment]
@kaikreuzer Just trying to setup the build for snapshot addons again. Is it right that there is no legacy-addons snapshot anymore?

**Comment by kaikreuzer** [comment]
>  Is it right that there is no legacy-addons snapshot anymore?

Legacy or 1.x add-ons in general? For legacy, yes, that is certain. For 1.x add-ons, there is no final decision yet, see https://github.com/openhab/openhab1-addons/pull/5935#issuecomment-583744300.

**Pull request merged**